osu.Game/Tests/Visual/ModSandboxTestScene.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Scoring;
+using osu.Game.Screens.Play;
+
+namespace osu.Game.Tests.Visual
+{
+    public abstract class ModSandboxTestScene : PlayerTestScene
+    {
+        protected sealed override bool HasCustomSteps => true;
+
+        public override IReadOnlyList<Type> RequiredTypes => new[]
+        {
+            typeof(ModSandboxTestScene)
```

Comment:
is this here as a requirement for inherited test scenes? sounds like something we may want to automate framework-side.